app/config.py

```diff
@@ -374,3 +374,5 @@ def sl_getenv(env_var: str, default_factory: Callable = None):
 ALIAS_LIMIT = os.environ.get("ALIAS_LIMIT") or "100/day;50/hour;5/minute"
 
 ENABLE_SPAM_ASSASSIN = "ENABLE_SPAM_ASSASSIN" in os.environ
+
+ALIAS_RAND_SUFFIX_LENGTH = int(os.environ.get("ALIAS_RAND_SUFFIX_LENGTH", 5))
```

Comment:
I wonder if we can use "RANDOM" instead of the abbreviated version "RAND" :)?

Reply:
Do you think 5 characters can be enough to avoid potential conflict?

Reply:
That's (26+10)^5=60,466,176 possible strings so I think it would be enough 🤔